From: Jason Rhinelander <jason@imaginary.ca>
Date: Mon, 18 Jan 2021 17:30:36 -0400
Subject: Re-vendor spdlog

(Don't merge this into the previous commit so that it can be used
everywhere, while this is only needed on systems with an older spdlog).
---
 common/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/CMakeLists.txt b/common/CMakeLists.txt
index ea24ba7..8c4cc90 100644
--- a/common/CMakeLists.txt
+++ b/common/CMakeLists.txt
@@ -6,6 +6,6 @@ add_library(common STATIC
     src/oxen_logger.cpp
 )
 
-find_package(spdlog)
+add_subdirectory(../vendors/spdlog spdlog)
 target_link_libraries(common PUBLIC spdlog::spdlog filesystem oxenmq::oxenmq)
 target_include_directories(common PUBLIC ${CMAKE_CURRENT_LIST_DIR}/include)
